client/notifications/blocks/block-tree-parser.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {
+	flip,
+	head,
+	propOr
+} from 'ramda';
+
+import CommentBlock from './comment-block';
```

Comment:
All of these are note-types? We should look at placing them as blocks in `client/blocks/note-{type}/index.jsx` each with their own styles and render them in devdocs.